<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analyzereport</class>
 <widget class="QDialog" name="analyzereport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BudgetBuddy - Report Analysis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: rgb(202, 219, 194);
   </string>
  </property>

  <!-- Title -->
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 18pt "Segoe UI"; color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Your Report Analysis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- Filter Controls -->
  <widget class="QLabel" name="labelFilter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt "Segoe UI"; color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Filter By:</string>
   </property>
  </widget>

  <widget class="QComboBox" name="comboBoxMonth">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt "Segoe UI"; color: black; background: white;</string>
   </property>
  </widget>

  <widget class="QSpinBox" name="spinBoxYear">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt "Segoe UI"; color: black; background: white;</string>
   </property>
  </widget>

  <widget class="QPushButton" name="buttonFilter">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt "Segoe UI"; color: black; background: white; border: 2px solid black; border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>

  <!-- Labels -->
  <widget class="QLabel" name="labelTotalIncome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 12pt "Segoe UI"; color: black;</string>
   </property>
   <property name="text">
    <string>Total Income:</string>
   </property>
  </widget>

  <widget class="QLabel" name="labelTotalExpenses">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>150</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 12pt "Segoe UI"; color: black;</string>
   </property>
   <property name="text">
    <string>Total Expenses:</string>
   </property>
  </widget>

  <widget class="QLabel" name="labelTotalMonthlyGoal">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 12pt "Segoe UI"; color: black;</string>
   </property>
   <property name="text">
    <string>Total Monthly Goal:</string>
   </property>
  </widget>

  <widget class="QLabel" name="labelRemainingBudget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 12pt "Segoe UI"; color: black;</string>
   </property>
   <property name="text">
    <string>Remaining Budget:</string>
   </property>
  </widget>

  <!-- Pie Chart Frame -->
  <widget class="QFrame" name="chartWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>700</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white; border: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>

  <!-- Hover Info Label -->
  <widget class="QLabel" name="labelHoverInfo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt "Segoe UI"; color: rgb(0, 85, 127);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>

  <!-- Warning Label -->
  <widget class="QLabel" name="labelWarnings">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>640</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt "Segoe UI"; color: black;</string>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- Back Button -->
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>680</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 12pt "Segoe UI"; color: black; background: white; border: 2px solid black; border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>Back to Homepage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
